### ISSUE TITLE
calculateMonthsDifference always returning the mont difference +1

### ISSUE THREAD

**Issue by vinsalmont** [adamstyrc/calendarro#15]
I was trying to use the date utils outside the calendarro, and the return of the difference between months is always returning the difference +1, maybe it is the + 1 at the end of the function? It was supposed to be there?

```
  static int calculateMonthsDifference(
      DateTime startDate,
      DateTime endDate) {
    var yearsDifference = endDate.year - startDate.year;
    return 12 * yearsDifference
        + endDate.month - startDate.month + 1;
  }
  
```

**Comment by adamstyrc** [comment]
Hi @vinsalmont 

Oh, you're totally right. I did this mistake as a hotfix to something else and didn't pay attention.
Just released v1.1.2 ;)

Enjoy!

**Comment by vinsalmont** [comment]
Thanks bro!
